### ISSUE TITLE
Correction on synonym of Tripleurospermum inodorum

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#2758]
Re: http://data.canadensys.net/vascan/taxon/3639

The synonym Matricaria maritima var. agrestis (Knaf) Wilmott should be listed as Matricaria maritima var. agrestis Wilmott, since Knaf's name, Dibothrospermum agreste, is a superfluous name, so cannot be used as a basionym.  ICBN 52.1

IPNI: Remarks: nom. superfl. for Matricaria inodora L. (1755)

**Comment by brouille** [comment]
Confirmation requested from K. Gandhi since the error also occurs in FNA
Luc

**Comment by brouille** [comment]
Here is the opinion expressed after a through reserach by Kanchi Gandhi (email 2018-03-21)
"Homotypic names (NOT complete) listed below:
1) Matricaria inodora L. (1755)
2) Matricaria inodora L. var. inodora (1852): created by M. inodora var. eligulata Seem. (1852)
3) Tripleurospermum maritimum subsp. inodorum (L.) Appleq. (2003): OK
4) Dibothrospermum agreste Knaf (1846): superfl. & illegit. (for no. 1)
5) Matricaria inodora L. var. agrestis Weiss (1895): invalid; includes the type of M. inodora
6) Matricaria inodora f. agrestis Fiori & Paol., Fl. Italia 2: 236, 1903: invalid; includes the type of M. inodora
7) Tripleurospermum maritimum var. agreste Briq. & Cavill. (1916): superfl., because the homotypic varietal epithet inodora was available (from 1852)
8) Chrysanthemum maritimum var. agreste Bech., Verh. Naturf. Ges. Basel 32: 198. 1921: superfl., because the homotypic varietal epithet inodora was available (from 1852)
9) Matricaria maritima var. agrestis Wilmott ex Fernald, Manual (Gray), ed. 8. 1517. 1950: superfl., because the homotypic varietal epithet inodora was available (from 1852)"

in conclusion, the name Matricaria maritima var. agrestis (Knaf) Wilmott will simply be removed from the synonymy list in VASCAN and not be replaced by anoter name, as all are superfl. or inval.

LB
